prototype/triton/README.md

```diff
@@ -0,0 +1,10 @@
+# `Triton` Quant
+
+### Composable `triton` kernels for quantization workflows
+
+#### TODO
+
+- Common quant / dequant kernels for popular quantization frameworks
+  - [ ] GPTQ
```

Comment:
We already have code for GPTQ https://github.com/pytorch-labs/ao/blob/main/torchao/quantization/GPTQ.py
Bits and bytes is also interesting granted we also have kernels for QLoRA that are codegened here as well
HQQ would be nice to add

Reply:
Ok -- will work on kernels for `bitsandbytes` `AdamW8bit` then onto HQQ.

HQQ's 4 and 8-bit should be easy to adapt; 1, 2, and 3-bit might require some additional preprocessing to optimize.

Their current CUDA dequant implementations can definitely be optimized.  Will work on re-implementing in `CUDA` and `triton`.